libs/application/templates/parental-leave/src/fields/Rights/GiveDaysSlider.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState } from 'react'
+import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { Controller, useFormContext } from 'react-hook-form'
+import { Box, Text } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import Slider from '../components/Slider'
+import BoxChart, { BoxChartKey } from '../components/BoxChart'
+
+const GiveDaysSlider: FC<FieldBaseProps> = ({ field, application }) => {
+  const maxDays = 45
+  const { id } = field
+  const { formatMessage } = useLocale()
+  const currentAnswer = getValueViaPath(
+    application.answers,
+    field.id,
+    undefined,
+  ) as number
+
+  const maxDaysToGive = 45
+
+  const { clearErrors } = useFormContext()
+
+  const [chosenGiveDays, setChosenGiveDays] = useState<number>(
+    currentAnswer || 1,
+  )
+
+  const daysStringKey = chosenGiveDays > 1 ? m.giveRightsDays : m.giveRightsDay
+
+  const yourRightsWithGivenDaysStringKey =
+    maxDaysToGive - chosenGiveDays === 1
+      ? m.yourRightsInMonthsAndDay
+      : m.yourRightsInMonthsAndDays
+
+  const boxChartKeys: BoxChartKey[] = [
+    {
+      label: () => ({
+        ...yourRightsWithGivenDaysStringKey,
+        values: { months: '5', day: maxDaysToGive - chosenGiveDays },
```

Comment:
You will probably want to move that `months: '5'` as well in the config file?

Reply:
Agree, should be based on the config now.